modules/programs/broot.nix

```diff
@@ -237,5 +245,27 @@ in {
 
     programs.nushell.extraConfig =
       mkIf cfg.enableNushellIntegration (shellInit "nushell");
+
+    programs.xonsh = mkIf cfg.enableXonshIntegration {
+      xonshrc = "xontrib load broot";
+      extraPackages = ps:
+        [
+          (ps.buildPythonPackage {
```

Comment:
This is probably not the way to do this. I don't know if home-manager allows it, but cramming it in here manually seems less than ideal.